packs/qualys/actions/launch_scan.yaml

```diff
@@ -0,0 +1,27 @@
+---
+name: launch_scan
+runner_type: python-script
+description: Launch a scan
+enabled: true
+entry_point: launch_scan.py
+parameters:  
+  title:
+    type: string
+    description: IP of the machine to scan
```

Comment:
Hm, the parameter name is `title`. I assume this is a label / title for a scan or something else? :)